README

```diff
@@ -1 +1,62 @@
-Presto Manager: A tool for managing Presto clusters.
+# Presto Manager
+
+A tool for managing Presto clusters. Targeted for Linux OS, not guarentee to work on Windows/MacOS.
+
+## Getting Started
+
+These instructions will get you a instance of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+- JDK 8, available at (http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+         Alternative  (http://openjdk.java.net/install/)
+- Python 2.4+, available at [python.org](https://www.python.org/downloads/)
+
+### Installing
+
+Clone the full distribution
+```
+git clone https://github.com/prestodb/presto-manager.git
+```
+
+## Compiling and Running
+
+Go the the directory you cloned the distribution
+```
+cd presto-manager
+```
+
+Use Maven  Wrapper to compile and generate executable JAR for the project
+```
+./mvnw clean install
+```
+
+### Executing
+
+```
+java -jar ./presto-manager-agent/target/AgentServer.jar
+```
+And
+
+```
+java -jar ./presto-manager-controller/target/ControllerServer.jar
+```
+These command will start the Agent and the Controller.
```

Comment:
Put this before you list the comments.
````markdown
These commands will start the Agent and Controller, respectively:
```
java -jar ./presto-manager-agent/target/AgentServer.jar
java -jar ./presto-manager-controller/target/ControllerServer.jar
```
````